new-frontiers-14/frontier-station-14
# Bluespace Jug

## About the PR



Adds a bluespace jug.  Stores 1000u, costs as much as a bluespace beaker, researched in the same tech, slots into chemical dispensers.

Minor point: fixed the tech icon for Advanced Anomaly Research to the CHIMP from the anomaly synchronizer (TODO: fix the comment).

Sprites could probably use some work, looks a bit like a watering can as-is.

## Why / Balance


Operation buff chemical dispensers is a go.  No stronger than a bluespace beaker, no larger, but fits into chemical dispensers and should allow for larger volumes of materials to be stored & poured without hassle.

Why the same volume?  Easier to work with the volumes and prices we already have.

## How to test


1. Research Bluespace Chemistry under Civilian Services, check the recipes in the medical techfab and protolathe, they should cost the same as bluespace beakers.
2. Print one, it should print an actual, usable bluespace jug.
3. Drink from a jug, insert it into a dispenser, should work like a jug.
4. Try to destroy it - it should take damage, but not destroy - this is similar to the bluespace beaker.

## Media


Bluespace jugs.  Both inhand and standing sprites have an unlit segment and a window to show approximate fill level (useful feature for jugs).  The moths are showing the various inhand sprites, the jugs on the floor show the fill levels working, and the machines show the various machines that can print the new jugs, along with blueprints for them.
![image](https://github.com/user-attachments/assets/d2fca6d8-8f7e-4007-8c3f-b2a62e1e68d4)

## Requirements



## Breaking changes


**Changelog**


:cl:
- add: Added 1000u bluespace jugs, unlocked through the Bluespace Chemistry tech.
